src/runtime_src/core/common/ishim.h

```diff
@@ -200,41 +201,11 @@ struct ishim
   ////////////////////////////////////////////////////////////////
 
 #ifdef XRT_ENABLE_AIE
-  virtual xclGraphHandle
-  open_graph(const xrt::uuid&, const char*, xrt::graph::access_mode am) = 0;
-
-  virtual void
-  close_graph(xclGraphHandle handle) = 0;
-
-  virtual void
-  reset_graph(xclGraphHandle handle) = 0;
-
-  virtual uint64_t
-  get_timestamp(xclGraphHandle handle) = 0;
-
-  virtual void
-  run_graph(xclGraphHandle handle, int iterations) = 0;
-
-  virtual int
-  wait_graph_done(xclGraphHandle handle, int timeout) = 0;
-
-  virtual void
-  wait_graph(xclGraphHandle handle, uint64_t cycle) = 0;
-
-  virtual void
-  suspend_graph(xclGraphHandle handle) = 0;
-
-  virtual void
-  resume_graph(xclGraphHandle handle) = 0;
-
-  virtual void
-  end_graph(xclGraphHandle handle, uint64_t cycle) = 0;
-
-  virtual void
-  update_graph_rtp(xclGraphHandle handle, const char* port, const char* buffer, size_t size) = 0;
-
-  virtual void
-  read_graph_rtp(xclGraphHandle handle, const char* port, char* buffer, size_t size) = 0;
+  virtual std::shared_ptr<graph_handle>
+  open_graph_handle(const xrt::uuid&, const char*, xrt::graph::access_mode)
```

Comment:
Can we bring this out of XRT_ENABLE_AIE

Reply:
@chvamshi-xilinx We should remove XRT_ENABLE_AIE,  I believe it is always enabled after sw_emu was changed to implement the AIE shim APIs on x64 (long time back).

Reply:
Sure @stsoe . That is in my TODO. We are using this in lot of places. will remove them in phases.